components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -3446,28 +3426,40 @@ def listOrphanedAnnotations(self, parent_type, parent_ids, eid=None,
         for e in q.findAllByQuery(sql, p, self.SERVICE_OPTS):
             yield AnnotationWrapper._wrap(self, e)
 
-    def countAnnotations(self, objects):
+    def getAnnotationCounts(self, objDict=None):
```

Comment:
empty dict in python is declared as `mydict= {}` or  https://docs.python.org/2/library/stdtypes.html

This will throw exception in a loop